lesson4.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <link rel="stylesheet" href="css/style.css">
+    <h1 class="heading">What do we offer</h1>
+    <hr>
+    <div class="container1">
+<img src="img/Vector.png" alt="image1" class="image1">
```

Comment:
classnames should be informative. Alts should contain information about what is in the picture